src/pages/ErrorPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Typography, Button } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { withRouter } from 'react-router-dom';
+
+import Navbar from '../components/Header/Navbar';
+import Footer from '../components/Footer/index';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1
+  },
+  bodyCopy: {
+    margin: '10rem',
+    textAlign: 'center'
+  },
+  link: {
+    color: theme.typography.h6.color,
+    margin: '1rem'
+  }
+});
+
+class ErrorPage extends Component {
+  handleBack = () => {
+    this.props.history.goBack();
+  };
+  render() {
+    const { classes } = this.props;
+    return (
+      <Grid container item xs={12} className={classes.root}>
+        <Navbar />
+        <Grid
+          container
+          item
+          xs={12}
+          direction="column"
+          justify="center"
+          alignItem="center"
+          className={classes.bodyCopy}
+        >
+          <Typography variant="h4">
+            404 Error: Oops! That Page could not be found!
+          </Typography>
+          <Typography variant="h5" className={classes.link}>
+            <Button variant="outlined" onClick={this.handleBack}>
```

Comment:
What does this button add? Doesn't the browser already provide back button?

Reply:
It does. A `Go Back` button provides the user with easy access to the previous page from the 404 page instead of having to use the browser.